mmedit/models/editors/nafnet/baseline_net.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmengine.model import BaseModule
+
+from mmedit.registry import MODELS
+from .local_utils import Local_Base
+from .utils import LayerNorm2d
+
+
+@MODELS.register_module()
+class Baseline(BaseModule):
```

Comment:
the class name Baseline is not suitable.

Reply:
Change it to NAFBaseline